src/components/dashboard/overview-section/stat-card/stat-card.tsx

```diff
@@ -0,0 +1,22 @@
+import { UserOutlined } from "@ant-design/icons";
+import { StatCardProps } from "../../../../@types/stats";
+import "./stat-card.css";
+
+export const StatCard = ({
+  icon = <UserOutlined />,
```

Comment:
why icon is coming from outside?

Reply:
I decided to allow the transfer of the Icon as a prop for flexibility and customization. This allows the component to choose which icon they want to display next to each statistic, thus making the component more versatile and dynamic.